eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -236,6 +203,10 @@ jobs:
 
                 Set-Content -Path $outputPath -Value $outputObject
 
+                $packageJsonDest = Join-Path -Path $(Build.ArtifactStagingDirectory)/packages "${{ artifact.Name }}"
+                $packageJsonSource = Join-Path -Path $(Build.ArtifactStagingDirectory) "packages/PackageInfo"
+                Copy-Item $packageJsonSource/${{ artifact.Name }}.json $packageJsonDest -Force
+
```

Comment:
What does this step do? Why do we need to run this step to get package version? Isn't this something that's already done by `Dump Package properties` or can we include the output from line 201-202 in the package info file created in previous step? What we need to do is to include the steps in Language-Setings.ps1 like other languages do.

Reply:
included in the language0settings.ps1